packages/flutter_tools/lib/src/commands/start.dart

```diff
@@ -132,10 +132,9 @@ class StartCommand extends StartCommandBase {
   Future<int> runInProject() async {
     logging.fine('Downloading toolchain.');
 
-    await Future.wait([
-      downloadToolchain(),
-      downloadApplicationPackagesAndConnectToDevices(),
-    ]);
+    // Load these in sequence so that errors common to them aren't printed twice.
```

Comment:
It's lame that we can't download these zip files in parallel.  Can we move the error checking to be up-front?